style.css

```diff
@@ -16,15 +16,9 @@ a {
 }
 
 h1 {
-    display: inline-block;
-    margin: 0;
     font-size: 2.125rem;
 }
 
-p {
-    font-size: 1rem;
-}
-
 @media screen and (min-height: 375px) {
     .center {
```

Comment:
🟡 Out of scope for this PR but another nice improvement

You could make this code a bit nicer to work with by using flex; absolute positioning is useful but kind of overkill for this case, and limits your ability to mess with padding & other positioning

![CleanShot 2024-01-31 at 19 20 10@2x](https://github.com/ReeceDrage/personal-website/assets/45462299/e791974e-2830-4685-83a8-91018d3e7b0c)